src/main/kotlin/io/curity/identityserver/plugin/signicat/signing/SigningClientFactory.kt

```diff
@@ -86,13 +86,12 @@ class SigningClientFactory
 
             if (trustManagers != null || keyManagers != null) {
                 sslContext.init(keyManagers, trustManagers, null)
-            }
```

Comment:
shouldn't we add an else statement here:
```
else
sslContext.init(null, null, null)
```
and then, below, set the SSL factory on the binding provider all the time (like before)?

Reply:
It works the way i did it so what is the point to set it when there is no trustmanagers? Either/or works for me

Reply:
Try to understand why it works and if we need to do anything else, is all. If it's working for you in both cases, I think that without that else case I mentioned, the context is essentially initialized that way (with the three null values). Was only concerned that `init` had to be explicitly called. If you found that it doesn't, then this change is OK with me.

Reply:
ah, now I see. If either of those is set, we create the context, like before. But now, we set this on the binding provider only when the trust manager and/or key manager is set. When neither of those is set, the binding provider must use an SSL context that is initialized with none.